tdrs-backend/tdpservice/search_indexes/admin/reparse_meta.py

```diff
@@ -8,18 +8,37 @@ class ReparseMetaAdmin(ReadOnlyAdminMixin):
 
     inlines = [DataFileInline]
 
+    def reparse_is_finished(self, instance):
+        """Overload instance property for ui checkboxes."""
+        return instance.is_finished
+    reparse_is_finished.boolean = True
+
+    def reparse_is_success(self, instance):
+        """Overload instance property for ui checkboxes."""
+        return instance.is_success
+    reparse_is_success.boolean = True
+
     list_display = [
         'id',
         'created_at',
         'timeout_at',
-        'success',
```

Comment:
we do lose filtering on these fields - we can create a custom filter for properties, future work

Reply:
@jtimpe couple questions --

- is this class really only for SSP files?
- is `True` the default value for is_finished and is_success? asking because i just ran the command on a large number of files and its still in the process of deleting records but these are the metrics im seeing ⬇️ 


![Screenshot 2024-10-11 112200](https://github.com/user-attachments/assets/c5b4f04b-9447-4f5f-9c97-2762e6833d2c)

Reply:
also, do we still need [these](https://github.com/raft-tech/TANF-app/blob/6ebee3e2d4a6a1504e949cd24ecfb3dab7ecd6b2/tdrs-backend/tdpservice/search_indexes/models/reparse_meta.py#L35L37)?

Reply:
@ADPennington 

> is this class really only for SSP files?

it should work for all file types. are you seeing something that specifies SSP only?

> is True the default value for is_finished and is_success? asking because i just ran the command on a large number of files and its still in the process of deleting records but these are the metrics im seeing

Yeah, this is interesting. So, under the hood running reparse first deletes all associated records, then creates the new associations. So while the new associations are not-yet-existing, the `all` in the `ReparseMeta` property might return true. I could update it to check how many file associations there are and respond accordingly.

> also, do we still need these

These fields are only used when reparse is run via the management command. we have #3205 which would refactor the action. we can remove the management command and these fields at that time.

Reply:
> @ADPennington
> 
> > is this class really only for SSP files?
> 
> it should work for all file types. are you seeing something that specifies SSP only?

just [this](https://github.com/raft-tech/TANF-app/blob/6ebee3e2d4a6a1504e949cd24ecfb3dab7ecd6b2/tdrs-backend/tdpservice/search_indexes/admin/reparse_meta.py#L1L7) 😄 

> > is True the default value for is_finished and is_success? asking because i just ran the command on a large number of files and its still in the process of deleting records but these are the metrics im seeing
> 
> Yeah, this is interesting. So, under the hood running reparse first deletes all associated records, then creates the new associations. So while the new associations are not-yet-existing, the `all` in the `ReparseMeta` property might return true. I could update it to check how many file associations there are and respond accordingly.

ok. worth noting, that once files were queued for parsing these booleans changed back to False. So a little misleading initially?


> > also, do we still need these
> 
> These fields are only used when reparse is run via the management command. we have #3205 which would refactor the action. we can remove the management command and these fields at that time.
ahh! got it.  thank you @jtimpe

Reply:
@ADPennington I pushed a change so that `is_finished` and `is_success` will stay `False` if there are no files queued. I also fixed the misleading docstring 😄

Reply:
k i'll re-deploy sometime over the weekend. lots of files currently reparsing in qasp